js/bmlt_tabs_admin.js

```diff
@@ -36,6 +37,7 @@ function numbersonly(myfield, e, dec)
 };
 
 jQuery(document).ready(function($) {
+	var tomato = "https://tomato.na-bmlt.org/main_server";
```

Comment:
can you set to `https://tomato.bmltenabled.org/main_server`

Reply:
nice will do, this is exact reason I wanted to add this feature. I cant even remember tomato url how are other people.